schemas/devContainer.base.schema.json

```diff
@@ -164,6 +164,55 @@
 					"type": "boolean",
 					"description": "Controls whether on Linux the container's user should be updated with the local user's UID and GID. On by default when opening from a local folder."
 				},
+				"containerEnv": {
+					"type": "object",
+					"additionalProperties": {
+						"type": "string"
+					},
+					"description": "Container environment variables."
+				},
+				"containerUser": {
+					"type": "string",
+					"description": "The user the container will be started with. The default is the user on the Docker image."
```

Comment:
> The default is the user on the Docker image

Is it possible that the default user is set by a Feature?

Reply:
Not currently. (The containerUser and remoteUser from the devcontainer.json will be included in the image metadata. So these are passed on from the base images in devcontainers/images.)